packages/wrangler/src/utils/print-bindings.ts

```diff
@@ -565,20 +572,43 @@ export function printBindings(
 			" (".length +
 			maxValueLength +
 			")".length;
+
+		const shouldWrap =
+			bindingLength + 6 + maxTypeLength + 6 + maxModeLength >=
```

Comment:
Where do these 6s come from?
Should they be computed from something to avoid them getting out of sync in the future?